### DISCUSSION TITLE
Insert new card mid-palette

### DISCUSSION THREAD

**Discussion by curtisbarnard** [primer/prism#31] [Ideas]
I think it would be nice to insert cards mid-palette. I've had two extent colors that I knew I wanted to use and then fill in the gaps between. Perhaps the '+' could add a new card to the right of whichever card is selected. For example where the arrow is below:

![image](https://user-images.githubusercontent.com/100104319/174218636-b5f0a62c-a760-4227-8449-47b0227450a3.png)

**Comment by colebemis** [comment]
Great idea. Thank you for the suggestion, @curtisbarnard!

**Comment by colebemis** [comment]
FYI, I'm converting this to a discussion in the "Ideas" category
